notebook/tests/selenium/test_buffering.py

```diff
@@ -0,0 +1,59 @@
+"""Tests buffering of execution requests."""
+
+from .utils import wait_for_selector
+
+
+def get_cell_text_output(notebook, index):
```

Comment:
Let's put `wait` somewhere in the name - I initially missed that this function was waiting for output to turn up.